src/helpers/actions/messages/replyToMessage.ts

```diff
@@ -142,6 +142,11 @@ export async function replyPrivately(
 	return message;
 }
 
+/**
+ * @deprecated Use {@link replyWithPrivateMessage} instead
```

Comment:
Ye

I plan to erase the deprecated functions in my next PR. I only left them deprecated because idk how far y'all have gotten, and whether you might've started referencing these yet.

Reply:
I got to it. Fixed